attributes/server.rb

```diff
@@ -1,21 +1,30 @@
 # Default attributes.
-node.default['rdiff-backup']['server']['start-hour'] = 5 # (9pm PST)
-node.default['rdiff-backup']['server']['end-hour'] = 13 # (5am PST)
-node.default['rdiff-backup']['server']['user'] = "rdiff-backup-server"
-node.default['rdiff-backup']['server']['sudo'] = true
-node.default['rdiff-backup']['server']['restrict-to-own-environment'] = true
-node.default['rdiff-backup']['server']['restrict-to-environments'] = []
-node.default['rdiff-backup']['server']['mailto'] = ""
-node.default['rdiff-backup']['server']['nagios']['alerts'] = true
-node.default['rdiff-backup']['server']['nagios']['plugin-dir'] = "/usr/lib64/nagios/plugins"
-node.default['rdiff-backup']['server']['jobs']['default']['destination-dir'] = "/data/rdiff-backup"
-node.default['rdiff-backup']['server']['jobs']['default']['exclude-dirs'] = []
-node.default['rdiff-backup']['server']['jobs']['default']['retention-period'] = "3M"
-node.default['rdiff-backup']['server']['jobs']['default']['additional-args'] = ""
-node.default['rdiff-backup']['server']['jobs']['default']['nagios']['alerts'] = true
-node.default['rdiff-backup']['server']['jobs']['default']['nagios']['max-change'] = 8192
-node.default['rdiff-backup']['server']['jobs']['default']['nagios']['max-late-start'] = 24
-node.default['rdiff-backup']['server']['jobs']['default']['nagios']['max-late-finish-warning'] = 4
-node.default['rdiff-backup']['server']['jobs']['default']['nagios']['max-late-finish-critical'] = 8
+default['rdiff-backup']['server']['start-hour'] = 5 # (9pm PST)
+default['rdiff-backup']['server']['end-hour'] = 13 # (5am PST)
+default['rdiff-backup']['server']['user'] = 'rdiff-backup-server'
+default['rdiff-backup']['server']['sudo'] = true
+default['rdiff-backup']['server']['restrict-to-own-environment'] = true
+default['rdiff-backup']['server']['restrict-to-environments'] = []
+default['rdiff-backup']['server']['mailto'] = ''
+default['rdiff-backup']['server']['nagios']['alerts'] = true
+default['rdiff-backup']['server'][
+  'nagios']['plugin-dir'] = '/usr/lib64/nagios/plugins'
+default['rdiff-backup']['server'][
+  'jobs']['default']['destination-dir'] = '/data/rdiff-backup'
+default['rdiff-backup']['server']['jobs']['default']['exclude-dirs'] = []
+default['rdiff-backup']['server'][
+  'jobs']['default']['retention-period'] = '3M'
+default['rdiff-backup']['server'][
+  'jobs']['default']['additional-args'] = ''
+default['rdiff-backup']['server'][
+  'jobs']['default']['nagios']['alerts'] = true
+default['rdiff-backup']['server'][
+  'jobs']['default']['nagios']['max-change'] = 8192
+default['rdiff-backup']['server'][
+  'jobs']['default']['nagios']['max-late-start'] = 24
+default['rdiff-backup']['server'][
+  'jobs']['default']['nagios']['max-late-finish-warning'] = 4
+default['rdiff-backup']['server'][
+  'jobs']['default']['nagios']['max-late-finish-critical'] = 8
```

Comment:
Couldn't this be simplified doing something like the following?

``` ruby
default['rdiff-backup']['server']['jobs']['default'] => {
  'exclude-dirs' => [],
  'retention-period' => '3M',
...
}
```

That way its easier to read.